blosc2/SChunk.py

```diff
@@ -274,7 +294,58 @@ def update_data(self, nchunk, data, copy):
         RunTimeError
             If some problem was detected.
         """
+        blosc2_ext._check_access_mode(self.urlpath, self.mode)
         return super(SChunk, self).update_data(nchunk, data, copy)
 
     def __dealloc__(self):
         super(SChunk, self).__dealloc__()
+
+
+def open(urlpath, mode="a",  **kwargs):
```

Comment:
Two spaces before `**kwargs`

Reply:
Fixed in 4b00b0a790da9cf1578b13a6cdf761552a28fb92.